docs/tasks/access-kubernetes-api/setup-api-aggregator.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Setup the API Aggregator
+assignees:
+- chenopis
+---
+
+{% capture overview %}
+
+Setting up the API Aggregator will allow the Kubernetes apiserver to be extended with additional APIs, which are not part of the core Kubernetes APIs. 
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* You need to have a Kubernetes cluster running.
+* Make sure there are Aggregator certificates (both CA and proxy) setup on the Kubernetes apiserver.
+* The proxy cert must have been signed by the relevant CA. 
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Enable apiserver flags
+
+The system needs to be enabled via the following apiserver flags; however, they may have already been taken care of by your provider.
+
+    --requestheader-client-ca-file=<path to aggregator CA cert>
+    --requestheader-allowed-names=aggregator
+    --requestheader-extra-headers-prefix=X-Remote-Extra-
+    --requestheader-group-headers=X-Remote-Group
+    --requestheader-username-headers=X-Remote-User
+    --proxy-client-cert-file=<path to aggregator proxy cert>
+    --proxy-client-key-file=<path to aggregator proxy key>
+
+If you are not running kube-proxy on your master, as the [Kubernetes Architectural Roadmap](https://docs.google.com/a/google.com/document/d/1XkjVm4bOeiVkj-Xt1LgoGiqWsBfNozJ51dyI-ljzt1o/edit?usp=sharing) recommends, then you need to make sure that the system is enabled with the following apiserver flag. Again, this may have already been taken care of by your provider.
```

Comment:
I found this a tad confusing. Just to be clear the roadmap recommends not running kube-proxy on the master.